### DISCUSSION TITLE
Want to confirm if I'm right about adding a icon

### DISCUSSION THREAD

**Discussion by dushmanta05** [simple-icons/simple-icons#8590] [General]
Hi there!
I want to contribute a icon (to use that in shields badge), and I've read the docs how to add a new icon, since I'm a newbie in open-source world I just wanted to ask you people if I'm right with the procedure of adding the icon.

I'm adding "Peerlist" icon So to add the icon I've to first import these below line in _data/simple-icons.json file

        {
            "title": "Peerlist",
            "hex": "#00AA45",
            "source": "https://peerlist.io/"
        }

and add the peerlist.svg file to icons folder, then create a PR to merge.
Is this the right procedure? Please do let me know, also can I add the .svg file as a color one?

Thank you in advance.

**Comment by jwr1** [comment]
You are pretty much correct. I will say though, your hex color field should not contain the `#` symbol at the front (it should look like `"hex": "00AA45"`), and the SVG has to be monochrome, so no colors.

**Reply by dushmanta05** [reply]
Thank you so much for your response. I have another question: the logo I want to add has three colours: white, black, and green. If I make it monochrome and give one hex code, then the logo will only be two colours, right? But I need three colours, so I'm a bit confused. Can you help me through?

**Reply by jwr1** [reply]
After looking at the logo, I'd say you should probably use the same color for the white and green sections.

The logo could probably really look like either of the following, depending on if you think it's important to keep the background square:

![peerlist-bg](https://user-images.githubusercontent.com/47087725/235454300-9bed84f6-fb41-4be8-8082-d9310d117cb4.png)
![peerlist-fg](https://user-images.githubusercontent.com/47087725/235454547-d38dea1b-fa23-4c76-b03b-17eef18d06a1.png)

The goal of converting icons to monochrome is to recreate the icon using only two colors (black & white, which is transparent) while keeping it recognizable. In the case of Peerlists's icon, making the green portion and the inside white portion the black part still allows for Peerlists's "P" to be identifiable.

**Reply by dushmanta05** [reply]
You're right; I think the square doesn't matter, so image 2 can be used. I was just a bit skeptical because the P logo would look like a normal P, so I wanted to use all 3 colors, but as you have mentioned, I can use the 2 logos with only 2 colors.

Thank you for answering my query and helping 😊, I really appreciate it.
